libs/api/domains/cms/src/lib/cms.mail.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common'
+import sgMail from '@sendgrid/mail'
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY)
```

Comment:
We might want to access this env variable through the environment file to maintain single point of entry for env variables?